frontends/election-manager/src/screens/logic_and_accuracy_screen.tsx

```diff
@@ -22,7 +21,12 @@ export function LogicAndAccuracyScreen(): JSX.Element {
           </p>
         ) : (
           <React.Fragment>
-            <ZeroReportPrintButton />
+            <p>
+              Go to the “Tally” tab and print the “Unofficial Full Election
+              Tally Report”. This report is referred to as the “Zero Report”
+              because, before CVRs have been imported, all tallies should be
+              zero.
+            </p>
```

Comment:
```suggestion
            <p>
              Go to the “Tally” tab and print the “Unofficial Full Election
              Tally Report”. This report is referred to as the “Zero Report”
              because — before CVRs have been imported — all tallies 
              should be zero.
            </p>
```

Reply:
I love a good use of an em dash! 😝